routes/users.route.ts

```diff
@@ -93,6 +96,48 @@ function parseQuery(query: UsersRequestQueryObj): UsersQueryObj {
   return usersQueryObj;
 }
 
+async function getUsersByQuery(req: Request, res: Response): Promise<void | undefined> {
+  const query = req.query;
+  const usersQueryObj: UsersQueryObj = parseQuery(query);
+  const limit = query && query.limit ? +query.limit : undefined;
```

Comment:
not good solution, that's mostly a trick, so use please `Number(query.limit)` instead of `+query.limit`